styles/themes/light.scss

```diff
@@ -183,6 +183,10 @@ html.light-mode {
 		}
 	}
 
+	.b-table .table-wrapper.has-mobile-cards .table {
+		background-color: #fff;
+	}
+
```

Comment:
```suggestion
```
we don't need this rule

Reply:
It's necessary to keep the same background color as on the pc web page.

You could find the difference from these pic.

<img width="357" alt="image" src="https://user-images.githubusercontent.com/31397967/185407495-0aeb9728-2349-4568-9cd4-88ef10420df4.png">

<img width="361" alt="image" src="https://user-images.githubusercontent.com/31397967/185407644-077b1273-cbb6-431e-bb33-77d399fe5a27.png">